### ISSUE TITLE
[a11y] Flash button in scanner is not accessible

### ISSUE THREAD

**Issue by SpacyRicochet** [openfoodfacts/openfoodfacts-ios#936]
The flash button in the scanner cannot be accessed by VoiceOver users, as it's not a proper accessibility element.

**Comment by teolemon** [comment]
I'm closing this one since you fixed it 🟢
